vector/src/main/res/values/strings.xml

```diff
@@ -3417,7 +3417,10 @@
     <string name="space_manage_rooms_and_spaces">Manage rooms and spaces</string>
 
 
-    <string name="labs_space_show_orphan_in_home">Experimental Space - Only show orphans in Home</string>
```

Comment:
Lint is not happy, you have to remove all the existing translations for this string.

Reply:
done